### ISSUE TITLE
$response.Headers.'X-Total-Count' Wrong Type in Invoke-ServiceNowRestMethod

### ISSUE THREAD

**Issue by natescherer** [Snow-Shell/servicenow-powershell#136]
Hello,
I am seeing a bug that appears to have been introduced with 2.3.1, it does not occur with 2.3.0.

# Environment

```
Operating System: Windows 10 1909
ServiceNow module version: 2.3.1 and 2.3.2
PowerShell version: 7.1.3
```

# Steps to reproduce

Run a cmdlet such as Get-ServiceNowTableEntry or Get-ServiceNowRecord

# Expected behavior

Cmdlet runs without error

# Actual behavior

```
InvalidArgument: C:\Users\nscherer\Documents\PowerShell\Modules\ServiceNow\2.3.1\Private\Invoke-ServiceNowRestMethod.ps1:170
Line |
 170 |      $totalRecordCount = [int]$response.Headers.'X-Total-Count'
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot convert the "System.String[]" value of type "System.String[]" to type "System.Int32".
```

**Comment by gdbarron** [comment]
Thanks @natescherer. Looks like ps V7 changed the headers so they are all arrays now. Fix coming soon.

**Comment by gdbarron** [comment]
fix in v2.4
